docs/relational-databases/partitions/partitioned-tables-and-indexes.md

```diff
@@ -63,7 +63,7 @@ An index that is built on the same partition scheme as its corresponding table.
 When partitioning a clustered index, the clustering key must contain the partitioning column. When partitioning a nonunique clustered index, and the partitioning column is not explicitly specified in the clustering key, [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] adds the partitioning column by default to the list of clustered index keys. If the clustered index is unique, you must explicitly specify that the clustered index key contain the partitioning column.        
 
 #### Partitioning NonClustered Indexes
-When partitioning a unique nonclustered index, the index key must contain the partitioning column. When partitioning a nonunique, nonclustered index, [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] adds the partitioning column by default as a nonkey (included) column of the index to make sure the index is aligned with the base table. [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] does not add the partitioning column to the index if it is already present in the index. 
+When partitioning a unique nonclustered index, the index key must contain the partitioning column. When partitioning a nonunique, nonclustered index on, [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] adds the partitioning column by default as a nonkey (included) column of the index to make sure the index is aligned with the base table. [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] does not add the partitioning column to the index if it is already present in the index.  Nonunique, nonclustered indexes on a table with a clustered index always include the clustered index key columns as nonclustered index key columns. 

```

Comment:
the addition of "on" in the first line is a typo

Reply:
While I understand why you'd want to add that generic info here, that info may be misunderstood as a characteristic of partitioned nonclustered indexes, and it's not. I'll add a note like "for more information on nonclustered indexes and index architecture, see https://docs.microsoft.com/en-us/sql/relational-databases/sql-server-index-design-guide?view=sql-server-ver15#Nonclustered"

Reply:
@davidbaxterbrowne Hi David, The link to nonclustered indexes and index architecture has been made to the documentation. Thank you for your contribution to Microsoft Docs.